cloud/jenkins/psmdb_operator_aks_version.groovy

```diff
@@ -0,0 +1,297 @@
+
+                        ./e2e-tests/$TEST_NAME/run
+                    fi
+                """
+            }
+            pushArtifactFile("$VERSION-$TEST_NAME-${params.PLATFORM_VER}-$MDB_TAG")
+            testsReportMap[TEST_NAME] = 'passed'
+            return true
+        }
+        catch (exc) {
+            if (retryCount >= 2) {
+                currentBuild.result = 'FAILURE'
+                return true
+            }
+            retryCount++
+            return false
+        }
+    }
+
+    echo "The $TEST_NAME test was finished!"
+}
+
+void conditionalRunTest(String TEST_NAME) {
+    if ( TEST_NAME == 'default-cr' ) {
+        if ( params.GIT_BRANCH.contains('release-') ) {
+            runTest(TEST_NAME)
+        }
+        return 0
+    }
+    runTest(TEST_NAME)
+}
+
+void installRpms() {
+    sh """
+        sudo yum install -y jq | true
+    """
+}
+pipeline {
+    parameters {
+        string(
+            defaultValue: 'main',
+            description: 'Tag/Branch for percona/percona-server-mongodb-operator repository',
+            name: 'GIT_BRANCH')
+        string(
+            defaultValue: 'https://github.com/percona/percona-server-mongodb-operator',
+            description: 'percona-server-mongodb-operator repository',
+            name: 'GIT_REPO')
+        string(
+            defaultValue: '1.21',
+            description: 'AKS kubernetes version',
+            name: 'PLATFORM_VER')
+        string(
+            defaultValue: '',
+            description: 'Operator image: perconalab/percona-server-mongodb-operator:main',
+            name: 'PSMDB_OPERATOR_IMAGE')
+        string(
+            defaultValue: '',
+            description: 'MONGOD image: perconalab/percona-server-mongodb-operator:main-mongod4.0',
+            name: 'IMAGE_MONGOD')
+        string(
+            defaultValue: '',
+            description: 'Backup image: perconalab/percona-server-mongodb-operator:main-backup',
+            name: 'IMAGE_BACKUP')
+        string(
+            defaultValue: '',
+            description: 'PMM image: perconalab/percona-server-mongodb-operator:main-pmm',
+            name: 'IMAGE_PMM')
+    }
+    environment {
+        CLEAN_NAMESPACE = 1
+    }
+    agent {
+         label 'docker'
+    }
+    options {
+        buildDiscarder(logRotator(daysToKeepStr: '-1', artifactDaysToKeepStr: '-1', numToKeepStr: '10', artifactNumToKeepStr: '10'))
+        skipDefaultCheckout()
+        disableConcurrentBuilds()
+    }
+
+    stages {
+        stage('Prepare') {
+            steps {
+                installRpms()
+                sh '''
+                    if [ ! -d $HOME/google-cloud-sdk/bin ]; then
+                        rm -rf $HOME/google-cloud-sdk
+                        curl https://sdk.cloud.google.com | bash
+                    fi
+
+                    source $HOME/google-cloud-sdk/path.bash.inc
+                    gcloud components update kubectl
```

Comment:
Do we need whole google-cloud-sdk, if we only use kubectl maybe we can just download that binary?